### ISSUE TITLE
How to deserialize an Update in a web server request handler?

### ISSUE THREAD

**Issue by madhead** [InsanusMokrassar/TelegramBotAPI#87]
I'm writing a TG webhook handler in Ktor. I guess, that in order to deserialize an `Update`, I need to use `UpdateSerializerWithoutSerialization` / `UpdateDeserializationStrategy` or have direct access to `RawUpdate` (which is serializable).

But all those classes are `internal` and cannot be used in user code. So, how do we deserialize incoming updates?

**Comment by InsanusMokrassar** [comment]
Hello! Here I pasted a minimal workaround for using of webhooks:

```kotlin
val bot = telegramBot("BOT TOKEN")
val flowUpdates = FlowsUpdatesFilter(64)
runBlocking {
    bot.setWebhook(
        "https://your.domain/some_secret",
        8081,
        flowUpdates,
        Netty
    )
}
```

Unfortunately, currently it is impossible to work with updates directly "by hands" and it is supposed that you will use long polling (via `startGettingOfUpdates`/`startGettingFlowsUpdates`) or webhooks (as it was mentioned in the example above, `setWebhook`). But I thikn, that it is possible to make `Update` serializable via setting up internal serializer. The problem here is that currently it does not support serialization. The other option is to make `UpdateDeserializationStrategy` public, but in this case I afraid that it could be a little bit sensitive to updates in other parts of library.

**Comment by madhead** [comment]
I think it's not a rare case when bot developers will use BotFather to set up webhooks for their own web servers written on Ktor or Spring Boot. Even more, some bots may use Heroku or AWS Lambda (serverless) deployments, where it's impossible to have a long-running process listening for updates, and where every update triggers (wakes up) some computation. It's crucial to have an ability to deserialize payloads directly to `Update` in such scenarios. Could you please consider making it possible?

**Comment by InsanusMokrassar** [comment]
Here `setWebhook` wll start ktor server for updates retrieving.  But I think that in the next release it will be allowed to deserialize updates outside of library.

**Comment by InsanusMokrassar** [comment]
Fixed in #88
